docs/en/changelogs/2.2.0.md

```diff
@@ -142,9 +142,9 @@ Sitemap.php
       * Translatable, for multilingual content
   * New third party libraries
       * PHP OpenID (http://openidenabled.com/php-openid/)
-      * Browscap (http://garetjax.info/projects/browscap/)
+      * Browscap (http://browscap.org/)
       * Plotr (http://www.solutoire.com/plotr/)
-      * SWFUpload (http://profandesign.se/swfupload/)
+      * SWFUpload (http://profandesign.se/)
```

Comment:
This link isn't relevant... either we should remove the link or leave it broken... or find a new more relevant one